tests/lib.rs

```diff
@@ -202,8 +203,9 @@ fn test_exchange_code_successful_with_minimal_json_response() {
             Url::parse(&(SERVER_URL.to_string() + "/token")).unwrap(),
         )),
     );
-    let token = client
-        .exchange_code(AuthorizationCode::new("ccc".to_string()))
+    let token = ::tokio::runtime::Runtime::new().unwrap().block_on(
```

Comment:
since each test needs to do this, let's define a function that takes a future, creates a runtime, blocks on it, and returns the result. it's probably worth trying to minimize the test boilerplate.